.github/labeler.yml

```diff
@@ -25,39 +25,39 @@ labels:
       above: 999
   - label: "airflow"
     files:
-      - "src/airflow/.*"
+      - "^src/airflow/.*"
   - label: "Documentation"
     files:
-      - "docs/.*"
+      - "^docs/.*"
   - label: "Dataset"
     files:
-      - "src/gentropy/dataset/.*"
+      - "^src/gentropy/dataset/.*"
   - label: "Method"
     files:
-      - "src/gentropy/method/.*"
+      - "^src/gentropy/method/.*"
   - label: "Datasource"
     files:
-      - "src/gentropy/datasource/.*"
+      - "^src/gentropy/datasource/.*"
   - label: "Step"
     files:
-      - "src/gentropy/[a-zA-Z]\\w+\\.py"
+      - "^src/gentropy/[a-zA-Z]\\w+\\.py"
   - label: "Feature"
-    title: "feat.*"
+    title: "^feat.*"
   - label: "Bug"
```

Comment:
I don't love this label. I think it makes sense when the title of the PR starts with `fix`, meaning that the changes intend to fix an existing problem.
For commit messages, I personally use `fix` quite often to address small fixes in the code I am developing (a typo, or test fixes). Unless I am using `fix` unconventionally, I'd omit this label.

Reply:
https://www.conventionalcommits.org/en/v1.0.0/#specification

Reply:
Typos are also a type of bug.

Reply:
do you prefer fix?

Reply:
Forget what I said. I thought labels were assigned from the commit messages, not the PR title. It makes sense like this. Sorry!